services/cypress/utils/apollo-test-utils.ts

```diff
@@ -0,0 +1,312 @@
+import { AxiosResponse } from 'axios'
+import {
+    ApolloClient,
+    DocumentNode,
+    HttpLink,
+    InMemoryCache,
+    NormalizedCacheObject,
+} from '@apollo/client'
+import { Amplify, Auth as AmplifyAuth, API } from 'aws-amplify'
+import { UnlockedHealthPlanFormDataType } from '../../app-web/src/common-code/healthPlanFormDataType'
+
+type StateUserType = {
+    id: string
+    email: string
+    givenName: string
+    familyName: string
+    role: 'STATE_USER'
+    stateCode: string
+}
+
+type CMSUserType = {
+    id: string
+    email: string
+    givenName: string
+    familyName: string
+    role: 'CMS_USER'
+    stateAssignments: []
+}
+
+type AdminUserType = {
+    id: string
+    email: string
+    givenName: string
+    familyName: string
+    role: 'ADMIN_USER'
+}
+
+type DivisionType = 'DMCO' | 'DMCP' | 'OACT'
+
+type UserType = StateUserType | AdminUserType | CMSUserType
+
+const contractOnlyData: Partial<UnlockedHealthPlanFormDataType> = {
+    stateContacts: [
+        {
+            name: 'Name',
+            titleRole: 'Title',
+            email: 'example@example.com',
+        },
+    ],
+    addtlActuaryContacts: [],
+    documents: [],
+    contractExecutionStatus: 'EXECUTED' as const,
+    contractDocuments: [
+        {
+            name: 'Contract Cert.pdf',
+            s3URL: 's3://local-uploads/1684382956834-Contract Cert.pdf/Contract Cert.pdf',
+            documentCategories: ['CONTRACT'],
+            sha256: 'abc123',
+        },
+    ],
+    contractDateStart: new Date('2023-05-01T00:00:00.000Z'),
+    contractDateEnd: new Date('2023-05-31T00:00:00.000Z'),
+    contractAmendmentInfo: {
+        modifiedProvisions: {
+            inLieuServicesAndSettings: false,
+            modifiedRiskSharingStrategy: false,
+            modifiedIncentiveArrangements: false,
+            modifiedWitholdAgreements: false,
+            modifiedStateDirectedPayments: false,
+            modifiedPassThroughPayments: false,
+            modifiedPaymentsForMentalDiseaseInstitutions: false,
+            modifiedNonRiskPaymentArrangements: false,
+        },
+    },
+    managedCareEntities: ['MCO'],
+    federalAuthorities: ['STATE_PLAN'],
+    rateInfos: [],
+}
+
+const newSubmissionInput = {
```

Comment:
I usually end up making this mock data into functions that return this info so that there's no chance of different tests modifying the global